test/e2e/run.sh

```diff
@@ -874,6 +874,34 @@ $py_assertion
     return 0
 }
 
+vm-set-kernel-cmdline() { # script API
+    # Usage: vm-set-kernel-cmdline E2E-DEFAULTS
+    #
+    # Adds/replaces E2E-DEFAULTS to kernel command line"
+    #
+    # Example:
+    #   vm-set-kernel-cmdline nr_cpus=4
+    #   vm-reboot
+    #   vm-command "cat /proc/cmdline"
+    local e2e_defaults="$1"
+    vm-command "echo 'GRUB_CMDLINE_LINUX_DEFAULT=\"\${GRUB_CMDLINE_LINUX_DEFAULT} ${e2e_defaults}\"' > /etc/default/grub.d/60-e2e-defaults.cfg" || {
```

Comment:
Have you tested that this works on a fedora VM, too ?

Reply:
No I tested only on Ubuntu. But I will verify on Fedora too.

Reply:
well it doesn't, welcome to Fedora :). I have no idea how to do it on Fedora since I don't use it. But I will do some search and try to find a way.

Reply:
On Fedora this is really easy using grubby.